lib/components/BuilderUI.jsx

```diff
@@ -200,12 +258,14 @@ export const BuilderUI = (props) => {``
   const [ticketSymbol, setTicketSymbol] = useState('')
   const [maxExitFeeMantissa, setMaxExitFeeMantissa] = useState('0.5')
   const [maxTimelockDuration, setMaxTimelockDuration] = useState('3600')
-  const [ticketCreditLimitMantissa, setTicketCreditLimitMantissa] = useState('0.1')
+  const [ticketCreditLimitMantissa, setTicketCreditLimitMantissa] = useState(
+    '0.1'
```

Comment:
same with this. Looks like sometimes it's moving useState args to new lines, and sometimes it's not?

Reply:
I think these are line length restrictions, we can adjust that